tools/workflow.sh

```diff
@@ -50,11 +51,13 @@ head $PEBBLES | awk '{print $1}' > sra_hits.txt
 
 
 ##############################################################################
-# TODO; Fetch fasta for the SRA hits, 
-#EB_QUERIES=FIXME
-#EB_QUERIES="/home/madden/CARIBOUFECES/caribou.query-list"
+# Fetch fasta for the SRA hits and deposit them in results bucket as queries for ElasticBLAST 
+bucket_no_prefix=$(echo $results_bucket | sed -e 's,gs://,,')
 parallel -t \
-    curl -s -X POST "https://us-east4-${ELB_GCP_PROJECT}.cloudfunctions.net/get_fasta" -H "Content-Type:application/json" -d '{"run_accession": "{}"}' -o {}.fsa
+    curl -X POST "https://${ELB_GCP_REGION}-${ELB_GCP_PROJECT}.cloudfunctions.net/get_fasta" -H "Content-Type:application/json" -d '{"run_accession": "{}", "upload_bucket": "$bucket_no_prefix", "upload_prefix": "fasta/"}' < sra_hits.txt
+gsutil ls $results_bucket/fasta > ${genbank}_hits.query-list
```

Comment:
clear `$results_bucket/fasta` before each run otherwise this will have results from previous runs

Reply:
Thanks, I created https://github.com/NCBI-Codeathons/psss-team4/pull/16 to address this.